src/MycilaESPConnect.cpp

```diff
@@ -670,7 +670,7 @@ void Mycila::ESPConnect::_startAP() {
   WiFi.setSleep(false);
   WiFi.persistent(false);
   WiFi.setAutoReconnect(false);
-  WiFi.softAPConfig(IPAddress(192, 168, 4, 1), IPAddress(192, 168, 4, 1), IPAddress(255, 255, 255, 0));
+  WiFi.softAPConfig(IPAddress(4, 3, 2, 1), IPAddress(4, 3, 2, 1), IPAddress(255, 255, 255, 0));
```

Comment:
why ?

Reply:
I've read some devices require the IP to be in the public space but is it still relevant or an old issue, I don't know.